### PR TITLE
Add Week 2 Photos and Markdown

### PR DESCRIPTION
**Pull request by mikegiunti**
It's Happening

### CODE REVIEW

File: Individual Assignments/Giunti Michele/week-02/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Biography
+
+## First Part (from week 1)
+
+My name is Michele Giunti, I am a 2nd year MPP student. I come from Rome, Italy and I have entered the program right after obtaining my undergraduate degree in Jacksonville, Florida.
+I have experience in academic research in both Psychology and Sociology, as well as some areas of communication, which was the source of my first published paper.
+During my undergraduate studies I was trained in SPSS, and had modules that included SAS, but I was only exposed to STATA by the time I started my graduate degree.
+While this time was short, I feel rather confident utilizing it, and I more recently learned to utilize R for both data visualization and data wrangling (mainly tidyverse and some experience with CRAN).
+My most recent research interest has involved urban planning and regeneration, with a concentration in architectural planning and business restoration within disadvantaged communities.
+However, I have opted to turn towards my psychology background in my thesis by engaging with the problem of loneliness among U.S. adults, considering the influence of the pandemic on social interactions.
```
Comment (Gustavomurillov) [inline Individual Assignments/Giunti Michele/week-02/README.md:10]: This sounds like a great thesis topic. Best of luck with that!

File: Individual Assignments/Giunti Michele/week-02/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+By putting research into the real world, we both bring attention the problem we are studying and we are also verifying the veracity of our results.
+
+It's a bit easier to do from me, since I've learned to approach people both linguistically (I learned 5 languages to compensate) and socially.
+Nevertheless, I understand that I need a lot more experience to implement my ideals, and I hope to gather such experience in this course.
+
+## Week 2 Section
+
+### Photos
+
+![Professional Picture]("img/picture_michele.png")
```
Comment (Gustavomurillov) [inline Individual Assignments/Giunti Michele/week-02/README.md:25]: Great pictures!

File: Individual Assignments/Giunti Michele/week-02/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+![My City]("img/rome.png")
+
+### Added Research Interests Paragraph
+
+As mentioned before, my current research revolves around the role of internet based relationships on loneliness among U.S. individuals.
+In fact, I distinguish types of loneliness depending on either being physically isolated with people (Social Isolation) from feeling isolated with people (Emotional Isolation).
+
+Loneliness has actually been a rather problematic phenomenon in the US, as shown in the table below ([KFF Report](https://www.kff.org/report-section/loneliness-and-social-isolation-in-the-united-states-the-united-kingdom-and-japan-an-international-survey-section-1/ "KFF Report")), with younger generations showing higher numbers of loneliness overall.
+
+|Age|Lonely|Not Lonely|
```
Comment (Gustavomurillov) [inline Individual Assignments/Giunti Michele/week-02/README.md:36]: This is a great table. However, I would suggest adding some blank spaces to make it straight lines (like a normal table) and make it look more aesthetically pleasant.

File: Individual Assignments/Giunti Michele/week-02/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+|18-49 NET|59%|52%|
+|18-29|24%|21%|
+|30-49|35%|31%|
+|50+ NET|41%|48%|
+|50-64|25%|26%|
+|65+|16%|21%|
+
+This is problematic not only on a mental health aspect, but also because of the several medical complications that loneliness might bring:
+
+<ol>
```
Comment (Gustavomurillov) [inline Individual Assignments/Giunti Michele/week-02/README.md:47]: I didn't use this code, and I don't see that it gets displayed anywhere, but I'm pretty sure it should work once posted? FYI: You can also make a list by using normal symbols such as numbers, hyphens, etc.

File: Individual Assignments/Giunti Michele/week-02/README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+<li>Cogntion Decline: Schizophrenia, Dementia, Alzheimer's</li>
+<li>Depression and Anxiety: HPA Inflammation, Suicidal Ideation and Attempts</li>
+<li>Chronic Health Conditions: Smoking, Drinking, Drug Use</li>
+</ol>
+
+My idea is to create a fixed effects equation for Health, Happiness and Trust againts levels of loneliness among U.S. americans. I do this by extracting data from the GSS and the ANES 2016-2020 longitudinal study.
+
+Here is some math (I can't preview it lmao).
+
+```math
```
Comment (Gustavomurillov) [inline Individual Assignments/Giunti Michele/week-02/README.md:58]: did you try $\sqrt{3x-1}+(1+x)^2$ ?

**Review by Gustavomurillov — APPROVED** [review]

File: Individual Assignments/Giunti Michele/week-02/README.md (hunk 4 above)
Comment (mikegiunti) [reply Individual Assignments/Giunti Michele/week-02/README.md:47]: It works on atom. It should also display on the readme.

File: Individual Assignments/Giunti Michele/week-02/README.md (hunk 3 above)
Comment (mikegiunti) [reply Individual Assignments/Giunti Michele/week-02/README.md:36]: What do you mean by this. Does the table not render on MD?

File: Individual Assignments/Giunti Michele/week-02/README.md (hunk 5 above)
Comment (mikegiunti) [reply Individual Assignments/Giunti Michele/week-02/README.md:58]: I am pretty sure this will be the result once I post the markdown

**Review by mikegiunti — COMMENTED** [review]
Let me know what the overall status is

**Comment by bbdaniels** [comment]
Hi @mikegiunti -- Make sure to check that everything works on web! As you can see the photo paths are not right.

@Gustavomurillov -- Same to you -- these are the types of things you should flag during review. Otherwise, great work on this to both of you!

Screen Shot

**Pull request merged**